### DISCUSSION TITLE
Style fish shell's vim mode prompt indicator in Oh My Posh

### DISCUSSION THREAD

**Discussion by thatanjan** [JanDeDobbeleer/oh-my-posh#4019] [Troubleshoot]
I use the fish shell with vim keybindings.
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/71136371/04c01914-c0a4-491c-a532-06d66eb265f9)

How can I style that indicator? I have tried to find the solution in docs, issues, etc. but could find anything useful.

Thanks in advance for the help

**Comment by JanDeDobbeleer** [comment]
@thatanjan that's a fish question, we do not print that indicator.
